modules/setting/mailer.go

```diff
@@ -93,6 +96,21 @@ func newMailService() {
 	MailService.FromName = parsed.Name
 	MailService.FromEmail = parsed.Address
 
+	switch MailService.EnvelopeFrom {
+	case "":
+		MailService.OverrideEnvelopeFrom = false
+	case "<>":
+		MailService.EnvelopeFrom = ""
+		MailService.OverrideEnvelopeFrom = true
+	default:
+		parsed, err = mail.ParseAddress(MailService.EnvelopeFrom)
+		if err != nil {
+			log.Fatal("Invalid mailer.ENVELOPE_FROM (%s): %v", MailService.EnvelopeFrom, err)
+		}
+		MailService.OverrideEnvelopeFrom = true
+		MailService.EnvelopeFrom = parsed.String()
```

Comment:
I think we can only use the `parsed.Address` here. Because RFC says that:

https://datatracker.ietf.org/doc/html/rfc5321#section-4.1.2

```
Syntax:
   mail = "MAIL FROM:" Reverse-path

Reverse-path   = Path / "<>"
Path           = "<" [ A-d-l ":" ] Mailbox ">"
```

But `parsed.String()` might return "name <a@b.com>".

Reply:
Agh! Yes you're right!